images/sk-opts.proto

```diff
@@ -26,9 +26,9 @@ message sk_opts_entry {
 	optional bool		so_reuseport	= 17;
 	optional bool		so_broadcast	= 18;
 	optional bool		so_keepalive	= 19;
-	optional uint32		tcp_keepcnt	= 20;
-	optional uint32		tcp_keepidle	= 21;
-	optional uint32		tcp_keepintvl	= 22;
+	optional uint32		tcp_keepcnt	= 20; /* obsolete */
+	optional uint32		tcp_keepidle	= 21; /* obsolete */
+	optional uint32		tcp_keepintvl	= 22; /* obsolete */
```

Comment:
Would you be able to add a comment like "deprecated in favour of tcp_opts_entry message"?

Reply:
How about I add a line of comment at the top that says "These three TCP options are deprecated, please use the tcp_opts_entry message instead"? and then there is no need to add "obsolete" to each one?

Reply:
The comment could be something similar to other deprecated flags:
```
$ ag deprecated
core.proto
44:	/* This is deprecated, should be per-thread */
49:	/* These two are deprecated, should be per-thread */

rpc.proto
88:	repeated criu_veth_pair		veths		= 19; /* DEPRECATED, use external instead */
94:	repeated ext_mount_map		ext_mnt		= 23; /* DEPRECATED, use external instead */
108:	repeated unix_sk                unix_sk_ino     = 33; /* DEPRECATED, use external instead */

```

Reply:
fixed